addons/account/models/account_analytic_default.py

```diff
@@ -60,6 +63,8 @@ def account_get(self, product_id=None, partner_id=None, account_id=None, user_id
             if rec.account_id: index += 1
             if rec.company_id: index += 1
             if rec.user_id: index += 1
+            if rec.analytic_id:
```

Comment:
You could inline this 
`if rec.analytic_id: index += 1`

Reply:
it created a style error https://runbot.odoo.com/runbot/batch/849857/build/19092187

Reply:
Alright, never mind then